### ISSUE TITLE
Add include '...' with {...} support

### ISSUE THREAD

**Issue by realcnbs** [mbosecke/pebble#104]
Such syntax should be allowed:
```
    {% include 'parts/text' with {
        'field': form.fields.get('url'),
        'addon': {
            'position': 'left',
            'text': 'Generate',
            'id': 'generate_post_url',
            'class': 'green'
        }}
    %}
```

**Comment by mbosecke** [comment]
:+1:

**Comment by thomashunziker** [comment]
I have create a pull request which contains this feature (see #109).

**Comment by mbosecke** [comment]
Resolved by 26d3040
